contracts/OgioParent.sol

```diff
@@ -29,26 +33,28 @@ abstract contract OgioService is AccessControl {
         Closed
     }
 
+    enum ContributionFrequency {
+        Weekly,
+        Bi_weely,
+        Monthly
+    }
+
     // Define the ROSCAGroup struct
     struct ROSCAGroup {
+        string groupId;
         string groupName;
-        string description;
```

Comment:
These text fields can be stored on a relational DB. Not a core necessity for the application.